.eslintrc.js

```diff
@@ -12,14 +11,11 @@ const options = {
 		'plugin:node/recommended',
 		'plugin:react/recommended',
 		'plugin:import/recommended',
-		'plugin:@typescript-eslint/recommended'
+		'plugin:@typescript-eslint/recommended',
+		'plugin:prettier/recommended'
 	],
 	parser: '@typescript-eslint/parser',
-	plugins: [
-		'react',
-		'import',
-		'@typescript-eslint'
-	],
+	plugins: ['react', 'import', '@typescript-eslint', 'prettier'],
```

Comment:
As you added `{
  "extends": ["plugin:prettier/recommended"]
}`
You do not need to specify 'prettier' in plugins.